osh/worker/tasks/task_get_analyzer_versions.py

```diff
@@ -17,7 +17,7 @@ class AnalyzerVersionRetriever(TaskBase):
     enabled = True
 
     # list of supported architectures
-    arches = ["noarch"]
+    arches = ["noarch", "aarch64", "x86_64"]
```

Comment:
Does it make sense to maintain a separate list of arches per each task class?  Could we rather move the list of supported arches to `worker.conf` so that OSH admins can introduce new arches without changing the code of OSH?

Reply:
That depends, now the settings can be more granular.  Nevertheless, such feature would require a modification in `kobo`.

On a second thought, such setting may be confusing.  It would not say what arches are supported by the `worker` itself, that is determined by the `hub`.  It only says that that the implementation of that task supports given arches in general, i.e. it has to be modified only if a new arch is introduces/removed to the whole service deployment.

Reply:
I still may change the `BaseTaskSettings` class name, though.

Reply:
What about naming it `AbstractTask` and letting it inherit from `TaskBase` so that we do not need to use multiple inheritance on all the leaf classes?

Reply:
Addressed in c6676c9.  Setting `enabled` is necessary so that worker does not see intermediate classes as available task types (the example uses a patched worker printing all registered types):
```sh
$ podman start -a osh-worker
ShutdownWorker TaskBase DiffBuild MockBuild VersionDiffBuild AnalyzerVersionRetriever ErrataDiffBuild
...
```